jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ProjectServiceRestController.java

```diff
@@ -75,17 +85,31 @@
         projectService.delete(projectId);
         return ResponseEntity.noContent().build();
     }
-
+    
+    @GetMapping(value = "/{projectId}/sessions/{sessionId}/report", produces = MediaType.APPLICATION_PDF_VALUE)
```

Comment:
incorrect endpoint mapping
+ reports should not belong to ProjectServiceRestController